test/helpers/balanceDiff.js

```diff
@@ -0,0 +1,10 @@
+async function balanceDifference (account, method) {
```

Comment:
nit: let's rename `method` to `promise`, to emphasize that it will be `awaited`.